### PR TITLE
include RSS feed

### LINKED ISSUE #53: RSS feed
There have been a couple of requests already for an RSS feed for the blog.
I do have an RSS feed for a Hugo static site that I could port (https://github.com/radu-matei/radu-matei.github.io/blob/content/themes/kiss/layouts/index.rss.xml), but I am by no means an expert here, so I would leave this to someone with more experience.

cc @flynnduism

### PR DESCRIPTION
**Pull request by flynnduism**
Adds links to the RSS feed (auto generated when Hugo builds) for the blog. I've placed it with the Blog category tags:

* Includes in the site `head` for reader detection
* Adds a link to the feed on `/posts`
* Adds a link to the feed on each blog post page

Closes #53.

---

Screen Shot 2021-05-17 at 2 25 16 PM

Screen Shot 2021-05-17 at 2 25 29 PM

Signed-off-by: flynnduism <dev@ronan.design>

### CODE REVIEW

**Comment by flynnduism** [comment]
https://deploy-preview-54--deislabsio.netlify.app/

Preview of the [blog feed .xml](https://deploy-preview-54--deislabsio.netlify.app/posts/index.xml).

**Review by carolynvs — APPROVED** [review]

File: themes/fresh/layouts/partials/css.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
 {{- $fontFace     := replace .Site.Params.font.name " " "+" }}
 {{- $fontSizes    := delimit .Site.Params.font.sizes "," }}
 {{- $fontUrl      := printf "https://fonts.googleapis.com/css?family=%s:%s" $fontFace $fontSizes }}
+<link href="{{ .Site.BaseURL }}posts/index.xml" rel="alternate" type="application/rss+xml" title="{{ .Site.Title }}" />
```
Comment (radu-matei) [inline themes/fresh/layouts/partials/css.html:8]: On the preview at least, this doesn't contain a trailing slash, resulting in `https://deploy-preview-54--deislabsio.netlify.appposts/index.xml`

File: themes/fresh/layouts/posts/list.html [hunk 2]
```diff
@@ -7,6 +7,7 @@
         {{ range $name, $taxonomy := .Site.Taxonomies.tags }}
           <li><a href="../tags/{{ $name | urlize}}">{{ $name }}</a></li>
         {{ end }}
+        <li><a href="{{ .Site.BaseURL }}posts/index.xml" class="rss"><i class="fa fa-rss"></i> Feed</a></li>
```
Comment (radu-matei) [inline themes/fresh/layouts/posts/list.html:10]: Same here re: the trailing slash.

**Comment by flynnduism** [comment]
I've updated the config for how deploy previews are generated, so the trailing slash issue is resolved. Site preview.
The RSS links will now generate as 

```
https://deploy-preview-54--deislabsio.netlify.app/posts/index.xml
```

rather than

```
https://deploy-preview-54--deislabsio.netlify.appposts/index.xml
```

**Review by radu-matei — APPROVED** [review]

**Pull request merged**